pkg/policyconditions/conditions.go

```diff
@@ -258,6 +201,88 @@ func update(apiWriter client.Client, apiReader client.Reader, policyReader clien
 	})
 }
 
+func setPolicyStatus(
+	policy *nmstatev1.NodeNetworkConfigurationPolicy,
+	nodeStatus *nodeStatusInfo,
+	logger *logr.Logger,
+) {
+	logger.Info(
```

Comment:
I think we could simply log this information before this function is called. Then we don't need the logger argument.